src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -72,9 +72,9 @@ The message bundle is attached to each panel component as the `parentBundle` opt
 Message bundles can be specified in one of two ways:
 
 1. through the [auxiliary schema](AuxiliarySchemaForPreferencesFramework.md) (if schemas are being used), or
-1. directly to the `messageLoader` (if grades are being used).
+2. directly to the `messageLoader` (if grades are being used).
```

Comment:
I'm thinking if we should remove grade version of preferences framework from this doc. I looked thru our other docs for preferences framework, the grade version is not mentioned elsewhere at all. Sort of remember we decided to promote schema version since it's more integrator friendly.